src/js/pages/Values/OneValue.jsx

```diff
@@ -264,20 +267,55 @@ class OneValue extends Component {
             <FilterChoices>
               <Chip
                 key="forThisElectionKey"
-                id = "forThisElection"
+                id="forThisElection"
                 label={<span style={showEndorsersForThisElection ? { fontWeight: 600 } : {}}>For This Election</span>}
                 className={showEndorsersForThisElection ? classes.selectedChip : classes.notSelectedChip}
                 component="div"
-                onClick={() => this.changeListModeShown('voterGuidesForThisElection')}
+                // Yousra Elzamzami - WV-1135
+                onClick={() => {
+                  const {pageName, pageType} = lookupPageNameAndPageTypeDict(window.location.pathname);
+                  TagManager.dataLayer({
+                    dataLayer: {
+                      event: 'filterToggleClick',
+                      filterSelected: 'voterGuidesForThisElection',
+                      pageDetails: {
+                        pageName,
+                        pageType,
+                        pathname: window.location.pathname,
+                      },
```

Comment:
In the [Parameters spreadsheet](https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090), I just created a new dictionary I think we should use here:
```
topicDetails: {
    topicName,
    topicWeVoteId,
    consideredLeft,
    consideredRight,
},
```
Please note that in the OneValue component, the "issue" prop is the same as "topic". We often use these two words interchangeably, and I think "topic" will be clearer within an analytics context.